### PR TITLE
Initial version of mentad integration

### PR DESCRIPTION
**Pull request by yoavyoav**
Been having an issue with the identify data not being added.

### CODE REVIEW

**Comment by f2prateek** [comment]
@yoavyoav Can you describe the issue in more detail? what identify data is not being added? Which line? What is the expected output and what is the actual output?

On a different note, I don't see why your integration needs to be on `analytics.js`. This could be a server side or direct integration, that accepts the same data, but is able to receive it from more sources.

**Comment by yoavyoav** [comment]
Oh yeah, sorry I wasn't clear.
We want a single call to the purchase tag once the order is completed,
the analytics.identify('id', {...}) call in the #completedOrder next to the
track call didn't work as
the analytics.deepEqual(window.mentad_shopper_info, ...) failed.
Regarding the other question - We don't require analytics.js, YourMechanic
requested it as they don't have the man power to allocate to enter our
simple enough server2server code.
After the time already spent I would prefer to finish the implementation,
If you guys will think this is still a long ways from ending then we will
consider it,
We would rather have a regular JS implementation YourMechanic and hopefully
in the future our other clients could just activate without
writing/implementing any code on the server side.

**Comment by yoavyoav** [comment]
Hello Prateek,
Do you have any ideas as to what is causing the empty dictionary and how can I push this issue forward?
Thanks!

File: lib/mentad/test.js [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+    describe('#completedOrder', function(){
+      beforeEach(function(){
+        analytics.stub(window.mentad_shopper_info, {});
+        analytics.stub(window.mentad_purchase_items, []);
+      });
+
+      it('should send ecommerce data', function(){
+        analytics.identify('id', {
+                            email: 'some@email.com',
+                            first_name: 'first',
```
Comment (f2prateek) [inline lib/mentad/test.js]: Our spec uses camel cased keys, you can see the documentation here https://segment.com/docs/spec/identify/#traits.

What this means is that you'd have to likely change this to:

```js
analytics.identify('id', { ..., firstName: '...', lastName: '...', ...});
```

**Comment by ndhoule** [comment]
This PR has gone stale so I'm closing this. If you'd like to get this going again, please reach out to our partners team at partners@segment.com. Thanks!

**Pull request closed without merging**